.env.sample

```diff
@@ -1,3 +1,4 @@
-ZKIT = 
-CIRCUIT_PATH = 
-NODE_ENV=
+ZKIT = "/home/xxx/workspace/ieigen/EigenZKit/target/release/zkit"
```

Comment:
donot have really assign the env variable here.  just let ZKIT = , so for others.